controllers/managedfusion_controller.go

```diff
@@ -466,10 +484,10 @@ func (r *ManagedFusionReconciler) reconcileAlertmanagerSecret() error {
 			return err
 		}
 		if r.pagerDutyConfigData.ServiceKey == "" {
-			return fmt.Errorf("Agent PagerDuty configuration does not contain serviceKey entry")
+			return fmt.Errorf("agent PagerDuty configuration does not contain serviceKey entry")
```

Comment:
- as we are touching these lines anyways, why not turn `fmt.Errorf` to `errors.New` if we aren't interpolating strings/errors?
- same for remaining

Reply:
We can. But I find it easier to use `fmt.Errorf` for consistency and to ease changes were we will want to introduce variables into the message

Reply:
- ack, I remember reading to use `errors.New` if there's no interpolation
- not that it gets in the same, so I don't feel any pressing need for the change